src/ducks/auth/sagas.js

```diff
@@ -21,7 +21,12 @@ function* authenticateSaga({ data }) {
 }
 
 function* logoutSaga() {
-  yield put(push('/login'));
+  try {
+    yield call(authService.logout);
+    yield put(push('/login'));
+  } catch (e) {
+    yield put(push('/login'));
```

Comment:
could `yield put(push('/login'));` live outside of the `try...catch`? It is fired regardless of the clause.